src/UI/Components/Diagram/interfaces.ts

```diff
@@ -21,6 +21,7 @@ interface DictDialogData {
 interface Rule {
   name: string;
   type: TypeEnum;
+  attrName?: string; //only used for inter-service relations
```

Comment:
overall, we try to avoid abbreviations, I believe it's a good practice to maintain.